janusgraph-core/src/main/java/org/janusgraph/core/JanusGraph.java

```diff
@@ -58,6 +58,37 @@
         reason = "JanusGraph test graph computer (FulgoraGraphComputer) " +
             "currently does not support graph filters but does not throw proper exception because doing so breaks numerous " +
             "tests in gremlin-test ProcessComputerSuite.")
+@Graph.OptOut(
+        test = "org.apache.tinkerpop.gremlin.process.computer.search.path.ShortestPathVertexProgramTest",
+        method = "*",
+        reason = "ShortestPathVertexProgram currently has two bugs that prevent us from using it correctly. See " +
+            "https://issues.apache.org/jira/browse/TINKERPOP-2187 for more information.")
```

Comment:
(nitpick) You could create a ticket in janusgraph to track it.

Reply:
A ticket to track another ticket? 😅 I can do that, but since we can't act on it until the issue is resolved in TinkerPop I would prefer to only create an issue in our issue tracker once it's resolved in TinkerPop. I also won't miss that as I created the ticket for TinkerPop and therefore will get a notification.

Reply:
I'm okay, if you track the other tickets.